### PR TITLE
Add Quick Start example; add Graphite to README

### PR DESCRIPTION
**Pull request by ramonza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,5 +256,6 @@ release in which the functionality was marked *Deprecated*.
 [exporter-stackdriver]: https://godoc.org/contrib.go.opencensus.io/exporter/stackdriver
 [exporter-zipkin]: https://godoc.org/go.opencensus.io/exporter/zipkin
 [exporter-jaeger]: https://godoc.org/go.opencensus.io/exporter/jaeger
-[exporter-xray]: https://github.com/census-instrumentation/opencensus-go-exporter-aws
+[exporter-xray]: https://github.com/census-ecosystem/opencensus-go-exporter-aws
```
Comment (odeke-em) [inline README.md]: Did you mean to prefix `https://godoc.org/` for these?

File: README.md [hunk 2]
```diff
@@ -255,5 +256,6 @@ release in which the functionality was marked *Deprecated*.
...
 [exporter-datadog]: https://github.com/DataDog/opencensus-go-exporter-datadog
+[exporter-graphite]: https://github.com/census-ecosystem/opencensus-go-exporter-graphite
```
Comment (odeke-em) [inline README.md]: Ditto about prefixing godoc.org

File: examples/quickstart/stats.go [hunk 3]
```diff
@@ -0,0 +1,165 @@
+// Copyright 2017, OpenCensus Authors
```
Comment (odeke-em) [inline examples/quickstart/stats.go]: s/2017/2018/g

File: README.md (hunk 1 above)
Comment (ramonza) [reply README.md]: No, I just changed the GH org.

File: README.md (hunk 2 above)
Comment (ramonza) [reply README.md]: I think the README is a better intro. They can always click through to the godoc.

**Pull request merged**